### ISSUE TITLE
sass-rails is deprecated

### ISSUE THREAD

**Issue by llekn** [platanus/potassium#201]
Generating a new project with potassium gives the following deprecation notice:

```
Ruby Sass has reached end-of-life and should no longer be used.

* If you use Sass as a command-line tool, we recommend using Dart Sass, the new
  primary implementation: https://sass-lang.com/install

* If you use Sass as a plug-in for a Ruby web framework, we recommend using the
  sassc gem: https://github.com/sass/sassc-ruby#readme

* For more details, please refer to the Sass blog:
  https://sass-lang.com/blog/posts/7828841

```

More info at https://github.com/rails/sass-rails/issues/420

**Comment by rjherrera** [comment]
It seems like the solution is to either replace `sass-rails` with `sassc-rails` or to upgrade the `sass-rails` gem to `~> 6.0`, which is a wrapper for `sassc-rails` as stated in https://github.com/rails/sass-rails/pull/424.

Either way, I can't find in the recipes where/when the gem is added to the gemfile. Any insights on this @llekn? ( 👋)

**Comment by llekn** [comment]
Hi @rjherrera 👋  !

Nice to know that `sass-rails` is wrapping `sassc-rails` for minimal disruption.

While I don't know exactly where is added, I do know that it seems that is a rails thing: I just ran `rails new --webpack=vue` (rails 5.2 and 6.0) and the `Gemfile` has `sass-rails` added.

Unfortunately even with rails 6 the `Gemfile` has `gem 'sass-rails', '~> 5'`.

Maybe we just need to wait this to be updated upstream?

**Comment by rjherrera** [comment]
Hi again @llekn!

I digged a little deeper and finally found that, as stated in https://github.com/rails/rails/issues/36954, it is no longer necessary to worry about this issue, because the `~> 5` version indicator will install the new `5.1.0` version released for this purpose, which is not raising any warnings or errors.

Also, in future releases of rails 6, starting with 6.0.1, the version of sass-rails will be bumped to `>= 6`. 

As we have confirmed that the issue will be solved and is not a potassium issue, I will close this issue 🎉.
